### DISCUSSION TITLE
How to make it wrap new line automatically by terminal width?

### DISCUSSION THREAD

**Discussion by leejack1000** [charmbracelet/bubbletea#555] [General]
Example:
`
func (m model) View() string {
	return fmt.Sprintf(
		"What’s your favorite Pokémon?\n\n%s\n\n%s",
		m.textInput.View(),
		"(esc to quit)",
	) + "\n"
}
`

<img width="225" alt="image" src="https://user-images.githubusercontent.com/11828473/196348940-2176d04c-6afa-4a99-96ed-8b95927d688a.png">

Expect:
What’s your favorite 
Pokémon?

**Comment by muesli** [comment]
You can use a `lipgloss.Style` to make views wrap at a particular width. In this particular case you can also use a `textarea`, which allows for multi-line input.

**Reply by leejack1000** [reply]
I see.  Thanks a lot.
